config/audit.php

```diff
@@ -25,12 +25,24 @@
 
     'implementation' => OwenIt\Auditing\Models\Audit::class,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Morphable
+    |--------------------------------------------------------------------------
+    |
+    | Set this to true to use a morphable user.
+    |
+    */
+
+    'morphable' => false,
```

Comment:
Move this to the `user` config down bellow, a line after `'model'       => App\User::class,`